packages/server/src/routes.ts

```diff
@@ -19,15 +19,28 @@ export const createRouter = (ctx: AppContext): express.Router => {
     res.send({ version })
   })
 
+  // @TODO paginate & test this
+  router.get('/export', async function (req, res) {
+    const fullExport = await ctx.db.fullExport()
+    res.setHeader('content-type', 'application/jsonlines')
```

Comment:
Ha, never seen the type for newline delimited json written this way!  I dig.

Reply:
yeah i gave it a good & this is what came up 🤷‍♂️